nilearn/plotting/surf_plotting.py

```diff
@@ -226,7 +229,7 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
 
     if surf_map is not None:
         surf_map_data = load_surf_data(surf_map)
-        if len(surf_map_data.shape) is not 1:
+        if len(surf_map_data.shape) != 1:
```

Comment:
please avoid changes that are not related to your PR

Reply:
This was flagged by flake8 and @emdupre [requested](https://github.com/nilearn/nilearn/pull/2477#pullrequestreview-422076318) I resolve the linting issues.

Reply:
this one was not pointed out by flake8 because it is only run on the diff with master. you can leave it if you prefer but in the future please remember to avoid changing lines that are not related to your PRs